pipelines/pyproject.toml

```diff
@@ -10,7 +10,7 @@ python-dotenv = "^0.19.0"
 fastapi = "^0.70.0"
 pydantic = "^1.8.2"
 psycopg2-binary = "^2.9.1"
-sqlalchemy = "^1.3.24"
+sqlalchemy = "~1.3.24"
```

Comment:
Let's bind to exact version `SQLAlchemy = "1.3.24"` 

Also, may I ask you to bind in all other services, lock and test?

Reply:
Now version of SQLAlchemy in all services is ==1.3.24, except scheduler, because of its tests: they fail with 1.3.24.